### PR TITLE
Remove Polygon refs

### PR DESCRIPTION
**Pull request by fmiguelgarcia**
Remove Polygon References

## TODO
- [x] File `SECURITY.md`

### CODE REVIEW

**Review by jakubcech — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ frame-executive = { path = "pallets/executive" }
 frame-system-rpc-runtime-api = { path = "pallets/system/rpc/runtime-api" }
 frame-system-benchmarking = { path = "pallets/system/benchmarking" }
 
-# Polygon Primitives
+# DA Primitives
 da-primitives = { version = "0.4", git="https://github.com/maticnetwork/avail-core", tag = "da-primitives/v0.4.5" }
```
Comment (prabal-banerjee) [inline Cargo.toml:28]: We need to change the cargo refs from `maticnetwork` repos to `availproject` repos, right?

File: node/Cargo.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 [package]
 name = "data-avail"
 version = "1.5.0"
-description = "Polygon Data Avilability Node."
+description = "Data Avilability Node"
```
Comment (prabal-banerjee) [inline node/Cargo.toml:4]: Should we change it to `Avail Node`? 
minor typo: `Avilability` -> `Availability`

File: node/src/command.rs [hunk 3]
```diff
@@ -31,7 +31,7 @@ use crate::{
 };
 
 impl SubstrateCli for Cli {
-	fn impl_name() -> String { "Polygon Avail Node".into() }
+	fn impl_name() -> String { "Data Availability Node".into() }
```
Comment (prabal-banerjee) [inline node/src/command.rs:34]: Does `Avail Node` make more sense?

File: runtime/Cargo.toml [hunk 4]
```diff
@@ -2,12 +2,12 @@
 name = "da-runtime"
 version = "6.0.0"
 authors = ["Anonymous"]
-description = "Polygon Data Avilability Runtime"
+description = "Data Avilability Runtime"
```
Comment (prabal-banerjee) [inline runtime/Cargo.toml:5]: Does `Avail Runtime` sound better? 
minor typo: `Avilability` -> `Availability`

File: Cargo.toml (hunk 1 above)
Comment (fmiguelgarcia) [reply Cargo.toml]: Done `maticnetwork` references were replaced.

**Pull request merged**